src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -38,28 +42,35 @@ export const getInstallations = async (client: GitHubAPI, subscriptions: Subscri
 	};
 };
 
-const getInstallation = async (client: GitHubAPI, subscription: Subscription, log?: Logger): Promise<AppInstallation> => {
-	const id = subscription.gitHubInstallationId;
+const getInstallation = async (client: GitHubAPI, subscription: Subscription, log: Logger): Promise<AppInstallation> => {
```

Comment:
You should be parsing which github client to use at a higher level and passing it to this function like we've done with the other PRs

Reply:
Did try originally but  we don't have the installation Id until we start mapping per subscription, so was allowing for different installation ids.

Could probably adjust the githubapp class to work but seems more effort than value at this point?

Reply:
ah right.